.eslintrc

```diff
@@ -109,20 +109,15 @@
       "rules": {
         "jsdoc/require-jsdoc": "off",
         "jsdoc/require-description": "off",
-        "tsdoc/syntax": "off",
-        "jest/expect-expect": [
-          "error",
-          {
-            "assertFunctionNames": ["expect", "**.should"]
-          }
-        ]
+        "tsdoc/syntax": "off"
       }
     },
     // Overrides for Cypress test files.
     {
       "files": ["**/integration/**/*.spec.{js,ts}"],
       "rules": {
-        "jest/valid-expect-in-promise": "off"
+        "jest/valid-expect-in-promise": "off",
+        "jest/expect-expect": "off"
```

Comment:
I've turned this off for all cypress tests because otherwise, we get errors from https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/expect-expect.md in many cypress tests.

Reply:
Nice. Although I am going to leave the rule for Jest tests :+1: